server/services/authService.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
```

Comment:
We don't need this here, if you use at the top of the index file it will be enabled globally